nntrainer/layers/addition_layer.cpp

```diff
@@ -45,6 +45,8 @@ void AdditionLayer::forwarding(bool training) {
   Tensor &hidden_ = net_hidden[0]->getVariableRef();
   TensorDim &in_dim = input_dim[0];
 
+  hidden_.setZero();
```

Comment:
good catch.
consider using `hidden_.add(net_input[0]->getVariableRef())` for the first iteration, and `hidden_.add_i(net_input[idx]->getVariableRef())` for the remaining inputs.

Reply:
Could you check https://github.com/nnstreamer/nntrainer/pull/1096#discussion_r612147251 and fix accordingly?

Reply:
Done